optimum/exporters/openvino/__main__.py

```diff
@@ -123,6 +125,19 @@ def main_export(
     >>> main_export("gpt2", output="gpt2_onnx/")
     ```
     """
+    if int8:
+        if fp16:
+            raise ValueError(
+                "Both `fp16` and `int8` were both set to `True`, please select only one of these options."
```

Comment:
is it really impossible to have both fp16 and int8 in the same time?
I think it should be possible

Reply:
wasn't sure this would work, should we set it by default to True for large models (along with `int8=True`) ? (enabling int8 weights and fp16 computations) ?

Reply:
enabling it back in  https://github.com/huggingface/optimum-intel/pull/443/commits/eefec6c064057111e17c77e60083d78594da5755

Reply:
from my knowledge, there are some potential accuracy issues with fp16 compression due to floating overflow for some models observable on GPU (like sometimes we see for stable diffusion models), so it may be dangerous to compress the model to fp16 by default. We are still investigating how to improve that and getting more info about which models are affected.